src/server/worker/driver/driver.go

```diff
@@ -559,18 +559,19 @@ func (d *driver) UserCodeEnv(
 			}
 		}
 	}
-	for _, input := range inputs {
-		result = append(result, fmt.Sprintf("%s=%s", input.Name, filepath.Join(d.InputDir(), input.Name, input.FileInfo.File.Path)))
-		result = append(result, fmt.Sprintf("%s_COMMIT=%s", input.Name, input.FileInfo.File.Commit.Id))
-		if input.JoinOn != "" {
-			result = append(result, fmt.Sprintf("PACH_DATUM_%s_JOIN_ON=%s", input.Name, input.JoinOn))
-		}
-		if input.GroupBy != "" {
-			result = append(result, fmt.Sprintf("PACH_DATUM_%s_GROUP_BY=%s", input.Name, input.GroupBy))
+	if len(inputs) > 0 {
```

Comment:
I don’t believe that this conditional does anything — if `len(inputs) == 0` then the `range` does nothing.